projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/src/serverRequests.ts

```diff
@@ -46,3 +46,25 @@ export function jobRunRequest() {
             );
         });
 }
+
+ /**
+    * Sent a POST request to the server to delete a data job.
+    * The information about the data job is retrieved from sessionStorage and sent as JSON.
+    */
+  export function deleteRunRequest() {
```

Comment:
method name seems confusing

Reply:
Yeah I agree, the naming here is v confusing